msal/src/main/java/com/microsoft/identity/client/AccountAdapter.java

```diff
@@ -82,7 +82,10 @@ public List<ICacheRecord> filter(@NonNull final List<ICacheRecord> records) {
             for (final ICacheRecord cacheRecord : records) {
                 final String acctHomeAccountId = cacheRecord.getAccount().getHomeAccountId();
                 final String acctLocalAccountId = cacheRecord.getAccount().getLocalAccountId();
-                if (acctHomeAccountId.contains(acctLocalAccountId)) {
+
+                // TODO: check the logic on broker side to make sure this value is NOT null.
```

Comment:
Is this TODO relevant?

Reply:
This change has gone away after merging with dev branch